contrib/upgrade_testnet/setup_genesis.py

```diff
@@ -44,9 +44,9 @@
     genesis['app_state']['auth']['accounts'] += chain_state['app_state']['auth']['accounts']
 
     # x/gov
-    genesis['app_state']['gov']['deposit_params']['max_deposit_period'] = '120s'
-    genesis['app_state']['gov']['voting_params']['voting_period'] = '120s'
-    genesis['app_state']['gov']['deposit_params']['min_deposit'] = [{'amount': '10000000', 'denom': 'udaric'}]
+    genesis['app_state']['gov']['params']['max_deposit_period'] = '120s'
+    genesis['app_state']['gov']['params']['voting_period'] = '120s'
+    genesis['app_state']['gov']['params']['min_deposit'] = [{'amount': '10000000', 'denom': 'udaric'}]
```

Comment:
The `x/gov` module now uses the `params` object instead of the various `x_params` fields